Scripts/script-misp_download_sample.yml

```diff
@@ -5,14 +5,26 @@ name: misp_download_sample
 script: |+
   import base64
 
```

Comment:
You should add release notes saying that this script is deprecated, and use the new command instead